tests/mcp/kubernetes/terraform/kubernetes_locals.tf

```diff
@@ -0,0 +1 @@
+/Users/dmitrogavrichuk/DEVOPS/Mesoform/terraform-infrastructure-modules/mcp/kubernetes_locals.tf
```

Comment:
This explicit path will cause tests to fail. Path should be relative.

Reply:
done